### PR TITLE
adds a 10x20mm packet

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds a 10x20mm packet, it's small.
it holds 2 and a half p90 magazines or whatever, 125 rounds.

## Why It's Good For The Game
10x20mm has needed a packet for a while
image

## Changelog
:cl:
expansion: 10x20mm packets added.
/:cl:

### CODE REVIEW

**Comment by Ieteihowmee** [comment]
Why do we not add speed loaders for magazines. They exist now to make magazine reloading so much easier, why not just add a little device you load/have preloaded with ammo? and in one click BOOM, mag is refeed.

**Comment by TiviPlus** [comment]
Balance freeze

**Comment by David-Stormwell** [comment]
where is my ammo packet tivi...

**Comment by SplinterGP** [comment]
Balans freeze gone

File: code/modules/projectiles/magazines/misc.dm [hunk 1]
```diff
@@ -113,3 +113,13 @@
...
 
+/obj/item/ammo_magazine/packet/p10x20mm
+	name = "packet of 10x20mm"
+	desc = "A packet containing 125 rounds of 10x20mm caseless.."
+	icon_state = "box_10x20mm"
+	caliber = CALIBER_10X20_CASELESS
+	default_ammo = /datum/ammo/bullet/smg
+	w_class = WEIGHT_CLASS_SMALL
+	current_rounds = 125
+	max_rounds = 125
```
Comment (BraveMole) [inline code/modules/projectiles/magazines/misc.dm:124]: Why 2.5 T90 magazines? why not 3?

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/magazines/misc.dm (hunk 1 above)
Comment (Pariah919) [inline code/modules/projectiles/magazines/misc.dm:124]:
```suggestion
	max_rounds = 150
```

File: code/modules/projectiles/magazines/misc.dm [hunk 2]
```diff
@@ -113,3 +113,13 @@
...
 	max_rounds = 100
 
+/obj/item/ammo_magazine/packet/p10x20mm
+	name = "packet of 10x20mm"
+	desc = "A packet containing 125 rounds of 10x20mm caseless.."
+	icon_state = "box_10x20mm"
+	caliber = CALIBER_10X20_CASELESS
+	default_ammo = /datum/ammo/bullet/smg
+	w_class = WEIGHT_CLASS_SMALL
+	current_rounds = 125
```
Comment (Pariah919) [inline code/modules/projectiles/magazines/misc.dm:123]:
```suggestion
	current_rounds = 150
```

**Review by BraveMole — APPROVED** [review]
ok

**Pull request merged**